crates/git_hosting_providers/src/providers/bitbucket.rs

```diff
@@ -258,14 +258,5 @@ mod tests {
             pr.url.as_str(),
             "https://bitbucket.org/zed-industries/zed/pull-requests/123"
         );
-
-        // Pull request number at end
-        let message = "Fix bug in authentication module (pull request #456)";
```

Comment:
@notpeter this is actually different from the previous one, previous one is the default message generated by BB, later one is a result of manual edit (the one we use), but agree in the end they are identical